capability/capability_test.go

```diff
@@ -68,3 +78,56 @@ func TestListSupported(t *testing.T) {
 		t.Fatalf("result is too short (got %d, want %d): +%v", len(list), minLen, list)
 	}
 }
+
+func TestAmbientCapSet(t *testing.T) {
```

Comment:
Hmm. You've just added `AmbientRaise`, `AmbientLower` and `AmbientClearAll` in the previous commit, but here you don't test those.

Reply:
Added one test now.